src/www/ui/ui-view-info.php

```diff
@@ -394,7 +394,7 @@ function ShowPackageInfo($Upload, $Item, $ShowMenu = 0)
           $entry = [];
           $entry['count'] = $Count;
           $entry['type'] = _($key);
-          $entry['value'] = htmlentities($R["$value"]);
+          $entry['value'] = $R["$value"];
```

Comment:
I dont think removing the sanitization will solve the issue. We introduced it because we want to preserve the html entities coming in `$R`. There is no other process defined which makes sure that the data is properly sanitized and will definitely create issues in escaping characters like `<` `>` and `&`.

If there is any other functionality in php packages that can solve it would be a better option. Something like: `htmlspecialchars`


Also please squash the commits :) and update the branch from base.